docker/docker-entrypoint.sh

```diff
@@ -11,7 +19,9 @@ done
 
 echo "PostgreSQL started"
 
+user_created=false
 #seed db
+# This seems SUPER weird. It returns true as long as DB exists; doesn't matter if admin@admin.com does.
 if psql --host=db --port=5432 --user=fourcat --dbname=fourcat -tAc "SELECT 1 FROM users WHERE name='admin@admin.com'"; then echo 'Seed present'; else
```

Comment:
I would prefer 'admin@localhost' or even just 'admin' (since we cannot use the e-mail address anyway): admin.com is an existing domain name so we should avoid using it

Reply:
(it is perfectly fine for users to not have an e-mail address as name, it's just not recommended since the e-mail is not stored elsewhere, so otherwise there is no contact information)